client/web/src/enterprise/search-jobs/SearchJobModal/SearchJobModal.tsx

```diff
@@ -0,0 +1,187 @@
+import { FC } from 'react'
+
+import { gql, useMutation } from '@sourcegraph/http-client'
+import { Button, ErrorAlert, H2, LoadingSpinner, Modal, Text } from '@sourcegraph/wildcard'
+
+import { SearchJobNode } from '../../../graphql-operations'
+import { SearchJobCard } from '../SearchJobCard/SearchJobCard'
+
+import styles from './SearchJobModal.module.scss'
+
+const DELETE_SEARCH_JOB = gql`
+    mutation DeleteSearchJob($id: ID!) {
+        deleteSearchJob(id: $id) {
+            alwaysNil
+        }
+    }
+`
+
+interface SearchJobModalProps {
+    searchJob: SearchJobNode
+    onDismiss: () => void
+}
+
+export const SearchJobDeleteModal: FC<SearchJobModalProps> = props => {
+    const { searchJob, onDismiss } = props
+
+    const [deleteSearchJob, { loading, error }] = useMutation(DELETE_SEARCH_JOB, {
+        onCompleted: (data, clientOptions) => {
+            // Delete just deleted search job from the apollo cache
+            clientOptions?.client?.cache.evict({ id: searchJob.id })
+            onDismiss()
+        },
+    })
+
+    return (
+        <Modal position="center" aria-label="Delete search job" onDismiss={onDismiss}>
+            <H2>Do you want to delete search job?</H2>
+
+            <Text className="mt-4">
+                <b>Note:</b> All query runs across all repositories will be stopped or canceled in case if the search
+                job is still running, search results will be deleted.
```

Comment:
This doesn't read well. Should it be this instead?
```suggestion
                <b>Note:</b> All query runs across all repositories will be stopped or canceled. In case if the search 
                job is still running search results will be deleted.
```